acceptance_tests/features/inbox_internal.feature

```diff
@@ -8,76 +10,99 @@ Feature: Internal inbox
     Given the internal user is already signed in
 
   @sm111_s01
+  @fixture.setup.data.with.unenrolled.respondent.user.and.internal.user
   Scenario: If there are no messages the user will be informed of this.
     Given the user has access to secure messaging
     And the user has no messages in their inbox
     When they navigate to the inbox messages
     Then they are informed that there are no messages
+    And the internal user signs out successfully
```

Comment:
Rather than adding this to all the features, perhaps we could do the sign out with a cleanup fixture? The context has an `add_cleanup` function where you can set scenario specific cleanup.

Reply:
fixed